source/change-streams/tests/unified/change-streams-resume-allowlist.yml

```diff
@@ -0,0 +1,1421 @@
+# Tests for resume behavior on server versions that do not support the ResumableChangeStreamError label
+description: "change-streams-resume-allowlist"
+schemaVersion: "1.7"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+      ignoreCommandMonitoringEvents: [ killCursors ]
+      useMultipleMongoses: false
+  - client:
+      id: &globalClient globalClient
+      useMultipleMongoses: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+  - database:
+      id: &globalDatabase0 globalDatabase0
+      client: *globalClient
+      databaseName: *database0
+  - collection:
+      id: &globalCollection0 globalCollection0
+      database: *globalDatabase0
+      collectionName: *collection0
+
+tests:
+  - description: change stream resumes after a network error
+    runOnRequirements:
+      - minServerVersion: "4.2"
+        topologies:
+          - replicaset
+          - sharded
+          - load-balanced
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: globalClient
```

Comment:
Note: you're using a string literal here when you likely intended to use a reference (i.e. `*globalClient`). I don't think it matters given what I'm about to say below but you may want to check for similar mistakes made elsewhere.

I didn't realize this in previous code reviews, but why do you use a global client here? This was never necessary in the original [POC tests I ported](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/valid-pass/poc-change-streams.yml).

Perhaps this was influenced by the [legacy spec test runner algorithm](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#spec-test-runner), but if it's not actually necessary I think we'd do well to drop this from the ported tests.

Reply:
Fixed the references here; see below for discussion of the client usage.